draft-ietf-ppm-dap.md

```diff
@@ -1230,7 +1227,7 @@ struct {
 
 struct {
   ReportShare report_share;
-  opaque payload<0..2^32-1>;
+  Message ping_pong_message;
```

Comment:
Hmm, we need to pick a more distinctive name than `Message`. Another option is to continue to handle this as an opaque byte string, including the length prefix. In fact, I think I'd prefer no to reference messages defined in another document.

Reply:
If we do that, then we should change the function definitions in VDAF so that instead of accepting and returning objects of type `Message`, they take opaque byte buffers. So instead of:

```
def ping_pong_continued(
            Vdaf,
            is_leader: bool,
            agg_param: bytes,
            state: State,
            inbound: Message,
        ) -> (State, Optional[Message])
```

We would have

```
def ping_pong_continued(
            Vdaf,
            is_leader: bool,
            agg_param: bytes,
            state: State,
            inbound: bytes,
        ) -> (State, Optional[bytes])
```

Which is less expressive, but keeps anything to do with `Message` inside of VDAF.

Reply:
Sounds like a reasonable solution to me.

Reply:
OK, I made this change, with a corresponding change on my VDAF PR (commit [2066205](https://github.com/cfrg/draft-irtf-cfrg-vdaf/pull/281/commits/2066205c572de1eb625ece322b9437d677948509) https://github.com/cfrg/draft-irtf-cfrg-vdaf/pull/281) to change the ping-pong interfaces.